opencensus/metrics/export/metric.py

```diff
@@ -0,0 +1,76 @@
+# Copyright 2018, OpenCensus Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from opencensus.metrics.export.metric_descriptor import MetricDescriptorType
+
+from opencensus.metrics.export.value import ValueLong
+from opencensus.metrics.export.value import ValueDouble
```

Comment:
Sort the imports, and import module instead of class.

Reply:
Ah I see the note in the style guide about class imports now, done.